### DISCUSSION TITLE
[Question]: Recommended RAM amount for Frigate

### DISCUSSION THREAD

**Discussion by iBobik** [blakeblackshear/frigate#10508] [Ask A Question]
### What is your question:

I just found the hard way Frigate with 4 fisheye cameras and Home Assiestant does not work on the old Raspberry Pi with 1GB RAM. What to buy for it?

There are **no mentions about needed RAM** amount: https://docs.frigate.video/frigate/hardware Just a few folks on this forum telling their NUC with 16 GB RAM is ok, but I does not want such overkill :-)

I like Odroid. Newest available is [M1](https://wiki.odroid.com/odroid-m1/odroid-m1) which offers 4 or 8 GB RAM.

Is 4GB for a whole system enought?
Is additional $35 for 8GB version worth it?

**Comment by NickM-27** [comment]
generally the problem isn't going to be available RAM but decoding power. https://docs.frigate.video/frigate/hardware#server

**Reply by iBobik** [reply]
So 4 GB should be enough for everything?

**Reply by NickM-27** [reply]
Should be fine for frigate
